README.yaml

```diff
@@ -24,7 +24,11 @@ categories:
   - kubernetes
   - aws
 
-description: Universal Tool for DevOps and Cloud Automation.
+description: |-
+  
+  `atmos` - Universal Tool for DevOps and Cloud Automation.
+  
+  ![atmos](docs/img/atmos-logo-128.png)
```

Comment:
Use the `logo` field. 

https://github.com/cloudposse/build-harness/blob/master/README.yaml#L11

And probably best to use SVG

Reply:
SVG is big, and I'm not sure how to correctly set the image size (especially in `logo` section`